requirements.txt

```diff
@@ -1,3 +1,9 @@
 Flask
-git+https://github.com/martel-innovate/wifi.git@rpi-fix
-python-wifi
\ No newline at end of file
+
+#In case of error, comment this line
```

Comment:
Why not leaving just the more "fixed" version git+https://github.com/martel-innovate/wifi.git@rpi-fix ?
there should be only one line per each module in the requirements.txt